node/clients/profile/profileV2.ts

```diff
@@ -308,13 +308,11 @@ export class ProfileClientV2 extends JanusClient {
   }
 
   public getUserPayments = (user: CurrentProfile, piiRequest?: PIIRequest) => {
-    const { userKey, alternativeKey } = this.getUserKeyAndAlternateKey(user)
     const url = this.getPIIUrl(
-      `${this.baseUrl}/${userKey}/purchase-info/unmask`,
-      alternativeKey,
+      `${this.baseUrl}/${user.userId}/purchase-info`,
```

Comment:
@beatrizmaselli com a remoção de `/unmask`, essa função se tornou basicamente igual à [getPurchaseInfo](https://github.com/vtex-apps/store-graphql/blob/6b549df7087f0aea2a7f5fa89b5b66a67dc9d68b/node/clients/profile/profileV2.ts#L86-L104).  
Sendo assim, não acha que faz mais sentido remover essa função e passar a usar diretamente `getPurchaseInfo`?

No mais, notei que `getPIIUrl` é uma função chamada por diversas rotas. Sabe dizer se a rota `/unmask` somente seria relevante para completar a rota de `/purchase-info` em `/purchase-info/unmask`?  

Em caso positivo, não acha que vale mais a pena extrair a lógica que faz a concatenação de `/unmask` de `getPIIUrl` e traze-la diretamente para a rota `getPurshaseInfo`. Evitariamos poluir a função com lógica que só faz sentido para uma das operações (`getProfileInfo`)
![image](https://github.com/vtex-apps/store-graphql/assets/172820829/387352d2-d921-4bcf-ae38-e44d340d99fd)

Em uma primeira olhada, me parece seguro fazer essa modificação. A concatenação só será feita se passada se for passado um `piiRequest` como você fez em `services.getPayments`.

Reply:
Não me parece o ideal porque o resolver para v1 e v2 é o mesmo e nesse caso para v1 a rota do getUserPayments é diferente da getPurshaseInfo.

Reply:
/unmask é usada para todas as rotas do profile para puxar as informações sensíveis abertas e não mascaradas.